security/advisories/index.html

```diff
@@ -23,6 +23,11 @@
                     </tr>
                 </thead>
                 <tbody>
+                    <tr>
+                        <td>May 5, 2023</td>
+                        <td><a href="{{ site.baseurl }}/security/advisories/2023-31047">CVE-2023-31047 Django file upload validation</a></td>
```

Comment:
This seems reasonable. Happy to have this instead of the title that I suggested.